content/en/blog/gitops-your-falco-rules/index.md

```diff
@@ -0,0 +1,805 @@
+`
+
+```
+- name: Build falcoctl
+  run: make
+  working-directory: tools/falcoctl
+```
+
+One last step before creating and uploading the OCI artifact is to have the rules we are pushing into it. This simple GitHub Action will download our repo into the `rules/` directory.
+
+```
+- name: Checkout Rules Repo
+  uses: actions/checkout@v3
+  with:
+    path: rules
+```
+
+#### Upload the OCI Artifact
+
+Here it is were the magic happens. This example spreads across multiple lines to make it easier to understand.
+
+```
+1| - name: Upload OCI artifacts to GitHub packages
+2|   run: |
+3|     tools/falcoctl/falcoctl registry push \
+4|     --config /dev/null \
+5|     --type rulesfile \
+6|     --version ${OCI_ARTIFACT_VERSION} \
+7|     ${OCI_REGISTRY}/${GITHUB_REPOSITORY}/${OCI_ARTIFACT_NAME}:${OCI_ARTIFACT_VERSION} \
+8|     rules/${RULESET_FILE}
+```
+
+The systax will look like: `falcoctl registry push [--option ...] destination-url source-file`. Let's dissect the command for a better understanding.
+
+- Line 2, uses a | symbol, which indicates that the all the segments at the same indentation level will be interpreted as a single line. YAML specifics.
+- Line 3, contains the **cli** verbs, the fresly compiled `falcoctl` tool with the subverbs `registry push` as shown above. The options will come later.
+- Line 4, `--config /dev/null`, is required to tell `falcoctl` not to use a configuration file. Otherwise it will try to create the directory `/etc/falcoctl` and we might not have permissions for that.
+Using a config file is an alternative, but for simplicity and pedagogical reasons we've kept everything on the command line.
+- Line 5, `--type rulesfile`, indicates what time of content should this artifact have. It could be a plugin or a rulesfile
+- Line 6, `--version ${OCI_ARTIFACT_VERSION}`, indicates the version of the artifact. It might look a bit redundant, but it's a mandatory option for the `falcoctl` command.
+- Line 7 is the destination URL to push the OCI artifact into. It should contain the registry, git user and repo name, name of the artifact, and the tag, which we have called version in this blog post.
+- Line 8 is just the name of the rulesfile that the OCI artifact will include.
+
+#### Credentials
+
+Our step won't need any previous authentication step thanks to the environment variable `FALCOCTL_REGISTRY_AUTH_BASIC` that we'll be adding at the end of the last step. This variable can contain any number of credentials separated by a semi-colon, but we require only one in our example. 
+
+The 3 elements of a credential, separated by commas, are the registry url, the user that is authenticating and the token for that user.
+
+Appended to the step where we were pushing the OCI artifact, and based on environmental information, we'll instantiate the variable as:
+
+```
+env:
+  FALCOCTL_REGISTRY_AUTH_BASIC: ${{ env.OCI_REGISTRY }},${{ github.repository_owner }},${{ secrets.GITHUB_TOKEN }}
```

Comment:
Note: using comma is strange, we more often use colons `:` or dots `.` (like the JWT)